src/ios/THEPeerBluetooth.m

```diff
@@ -631,7 +628,8 @@ - (void)peripheral:(CBPeripheral *)peripheral
         if ([[characteristic UUID] isEqual:_peerIDType])
```

Comment:
FIX AFTER CHECKIN - On line 362 there is peripheralManagerDidStartAdvertising handler but it doesn't look for, log, throw or do anything else with an error. That seems rather bad. Same with line 368 and peripheralManager.

And while we are in the area it would be good if we can update this file in general so it uses your dictionary object rather than the manual mutex locking for managing things like _peripherals.